htmlSanityCheck-core/src/test/java/org/aim42/htmlsanitycheck/test/dns/CustomHostNameResolver.java

```diff
@@ -21,7 +33,7 @@ public class CustomHostNameResolver {
 
     public InetAddress[] resolve(String hostname) throws UnknownHostException {
         // Custom DNS resolution logic
-        if (WIREMOCK_HOST.equals(hostname)) {
+        if (hostnames.contains(hostname)) {
```

Comment:
I guess WIREMOCK_HOST is no longer used then? Please remove it from the class if this is the case?

Reply:
I have removed WIREMOCK_HOST in this commit already can you check again ?

Reply:
You are right, sorry, forget my comment please!